app/models/subscription.rb

```diff
@@ -1,5 +1,5 @@
 class Subscription < ApplicationRecord
   validates :delivery_day, :subscription_type, presence: true
   belongs_to :user
-  has_many :shipments
+  has_many :shipments, dependent: :destroy
```

Comment:
Not sure if I'll allow users to delete their subscriptions/shipments because the admin may want to keep a record of these